### ISSUE TITLE
Code block meta string

### ISSUE THREAD

**Issue by bradwoods** [kentcdodds/mdx-bundler#84]
Is there anyway to access code block meta strings? Example:

in an mdx page - 

\```js myVar=myValue

const foo = "bar"
...

\```

[Reference](https://v0.mdxjs.com/guides/live-code)

**Comment by deadcoder0904** [comment]
@bradwoods Yes, there is. Check https://ped.ro/blog/code-blocks-but-better & the accompanying GitHub repo -> https://github.com/peduarte/ped.ro

Specficially -> https://github.com/peduarte/ped.ro/blob/main/lib/rehype-meta-attribute.js

As `mdx-bundler` uses `xdm`, you can read https://github.com/wooorm/xdm#syntax-highlighting-with-the-meta-field
